packages/commons/src/events/EventMetadata.ts

```diff
@@ -165,7 +165,8 @@ export const EventMetadata = z.object({
     .describe(
       'System-generated tracking ID used by informants or registrars to look up the event.'
     ),
-  flags: z.array(Flag)
+  flags: z.array(Flag),
+  modifiedAt: z.string().optional()
```

Comment:
is this `modifiedAt` required? Isn't this similar to `copiesPrintedForTemplate` in that we only supply it to the pdf?